docs/public/s2i-to-devfile.adoc

```diff
@@ -0,0 +1,92 @@
+== Migrating from S2I components to Devfile components
```

Comment:
I think we should add (Source-to-Image) in brackets next to S2I similar to the official docs: https://docs.openshift.com/enterprise/3.0/creating_images/s2i.html#:~:text=Source%2Dto%2DImage%20(S2I,ease%20of%20use%20for%20developers.